src/main/java/io/jenkins/plugins/ml/utils/ConvertHelper.java

```diff
@@ -51,17 +52,13 @@ public static String jupyterToText(FilePath jupyterFile) throws IOException, Int
         // Convert Jupyter Notebook to JSON
         try (final InputStreamReader inputStreamReader = new InputStreamReader(jupyterFile.read(), Charset.forName("UTF-8"))) {
             Note n = new JupyterUtil().getNote(inputStreamReader, "python", "\n", "#");
-            Gson gson = new Gson();
-            JsonElement tree = gson.toJsonTree(n);
-            JsonObject obj = tree.getAsJsonObject();
-            JsonArray array = obj.get("paragraphs").getAsJsonArray();
             StringBuilder outText = new StringBuilder();
-            for (JsonElement element : array)
-                if (element.isJsonObject()) {
-                    JsonObject cell = element.getAsJsonObject();
-                    String code = cell.get("text").getAsString();
-                    outText.append(code);
-                }
+            for (Paragraph para : n.getParagraphs()) {
+                System.out.println(para.getText());
```

Comment:
Need to remove the `System.out`, and either use the `Listener`, a `Logger`, or don't log. Otherwise we may clutter logs that are normally managed by sysadmins/devops/etc.

Reply:
Removed `System.out`. How do I vanish third party logs ?